node/authorship/src/tests.rs

```diff
@@ -794,14 +789,14 @@ fn block_builder_cloned_ok() {
     );
 }
 
-#[test]
+// Test case 6
```

Comment:
I'd name the functions than like `fn test_blah_blah`

Reply:
I didn't do that just in case we want (for debug purpose, for example), run each test individually with `cargo test` again. Such renaming would be an overkill.
But if we don't then yes, maybe it'd make sense to have uniform naming like `test_...`

Reply:
I guess remove prefix from 10fns isn't that hard if one day we return to `#[test]`